index.js

```diff
@@ -43,17 +38,32 @@ module.exports = (robot, options, visit) => {
     });
   }
 
+  // Trigger the event
+  function trigger(event) {
+    robot.webhook.emit(event.event, event);
+  }
+
   function schedule(installation, repository) {
     // Wait a random delay to more evenly distribute requests
     const delay = options.delay ? options.interval * Math.random() : 0;
 
     robot.log.debug({repository, delay, interval: options.interval}, `Scheduling interval`)
 
     setTimeout(() => {
-      // Schedule visit to this repository on an interval
-      intervals[repository.id] = setInterval(() => visit(installation, repository), options.interval);
-      // Make the first visit now
-      visit(installation, repository);
+      const event = {
```

Comment:
this should include `github` as a property as well. Currently we have to create a new github implementation within the callback based on the integration. Much like normal events provide this, the scheduled-events should include them as well.